glide.yaml

```diff
@@ -15,6 +15,7 @@ import:
   version: ^1
 - package: go.uber.org/thriftrw
   version: ^1
+- package: github.com/gogo/protobuf
```

Comment:
Yeah, since we dont know if `0.5` will bring breaking changes, we should pin to the minor here: `~0.4`